src/meshes/abstract_tree.jl

```diff
@@ -387,13 +387,17 @@ function refine!(t::AbstractTree, cell_ids,
     return refined_original_cells
 end
 
+@inline function coordinate_min_max_check(coordinates_min, coordinates_max)
+    for dim in eachindex(coordinates_min)
+        @assert coordinates_min[dim]<coordinates_max[dim] "coordinates_min[$dim] must be smaller than coordinates_min[$dim]!"
```

Comment:
The last min needs to be a max.

Reply:
Done in [02774b9](https://github.com/trixi-framework/Trixi.jl/pull/2257/commits/02774b9c74102a8dde629034268e15f30b373394).